src/containers/PatientDetails/PatientOverviewDynamic/index.tsx

```diff
@@ -0,0 +1,29 @@
+import { Patient } from 'fhir/r4b';
+
+import { Dashboards } from 'src/components/Dashboard';
+import { useDashboard } from 'src/containers/PatientDetails/PatientOverviewDynamic/hooks';
+
+import s from './PatientOverview.module.scss';
```

Comment:
I think it is better to use styled-components here instead of CSS modules.
@vesnushka could you please confirm?